packages/transitive-overlay/src/util.ts

```diff
@@ -177,6 +177,29 @@ function makeStop(stop: Place, coordinate?: number[]) {
     : stop;
 }
 
+/**
+ * Helper function that clones the 'to' stop and applies flex logic
+ * Flex routes sometimes have the same from and to IDs, but
+ * these stops still need to be rendered separately!
+ */
+function makeStopWithFlexSupport(leg: Leg): Place {
+  const flexSupportStop = { ...leg.to };
```

Comment:
This is not abstract enough. Can the function clone any place its given, and then have it accept `leg.to`?

Reply:
@miles-grant-ibigroup I thought about suggesting this change, however there is a call to `isFlex(leg)` in the function's body.

Reply:
Hmm that's right. Could we then rename this method to `renameLegFlexStops`?

Reply:
Yes, I'll do that right away!